api/migrations/20211012181841-createContributions.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        return queryInterface.createTable('Contributions', {
+            id: {
+                type: Sequelize.DataTypes.INTEGER,
+                primaryKey: true,
+                autoIncrement: true,
+                allowNull: false
+            },
+            contributor_id: {
+                type: Sequelize.DataTypes.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'Contributors',
+                    key: 'id'
+                }
+            },
+            issue_id: {
+                type: Sequelize.DataTypes.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'Contributors',
+                    key: 'id'
+                }
+            },
+            is_author: {
+                type: Sequelize.DataTypes.INTEGER,
+                allowNull: false
+            },
+            is_assigned: {
+                type: Sequelize.DataTypes.INTEGER,
+                allowNull: false
+            },
+            created_at: {
```

Comment:
Should this be consistent with `https://github.com/setlife-network/trinary/pull/543/files#diff-2f372035e9c231236cc776cd9644d3ea31edd29957006ea0981fd0094f572286R40`? same for `updated` field